osc-staging.py

```diff
@@ -146,35 +141,6 @@ def do_staging(self, subcmd, opts, *args):
 
     "check_duplicate_binaries" list binaries provided by multiple packages
 
-    "config" will modify or view staging specific configuration
-
-        Target project OSRT:Config attribute configuration applies to all
-        stagings. Both configuration locations follow the .oscrc format (space
-        separated list).
-
-        config
-            Print all staging configuration.
-        config key
-            Print the value of key for stagings.
-        conf key value...
-            Set the value of key for stagings.
-        config --clear
-            Clear all staging configuration.
-        config --clear key
-            Clear (unset) a single key from staging configuration
-        config --append key value...
-            Append value to existing value or set if no existing value.
-
-        All of the above may be restricted to a set of stagings.
-
-        The staging configuration is automatically cleared anytime staging
-        psuedometa is cleared (accept, or unstage all requests).
-
-        The keys that may be set in staging configuration are:
-
-        - repo_checker-binary-whitelist[-arch]: appended to target project list
-        - todo: text to be printed after staging is accepted
```

Comment:
Not only losing config command, but requested todo functionality. :(

Reply:
Right. And if you read the commit log of 0dc2769f443735ca0904c8cef5c15b2dea4d3587 I checked with staging managers that they want it differently anyway.

Reply:
The other use-case was "lost" during `git mv ...`. If the functionality is no longer desired should just be dropped as not relevant to "new staging api".

Reply:
You mean to cherry pick the drop of feature in another PR? Could be done - this is a draft